packages/components/src/theme-type-badge/premium-badge/index.tsx

```diff
@@ -55,7 +55,7 @@ const PremiumBadge = ( {
 	// so that the label can be measured in its uncompacted state.
 	const [ mayRenderAsCompact, setMayRenderAsCompact ] = useState( false );
 
-	const labelText = customLabelText || __( 'Premium' );
+	const labelText = customLabelText || __( 'Upgrade' );
```

Comment:
I think this might be misleading for sites that have already the Premium and up plans. When the site has Business, we still display an "Upgrade" badge even though the site is eligible for a Premium theme.

Reply:
It's also inconsistent with the other theme badges. For Woo and Partner we are still displaying the "WooCommerce" and "Partner" badges, but for Premium we will display an "Upgrade" badge.

Reply:
Yeah, you're right. My initial thought was to try to avoid using the premium badge to avoid implementing more badge tiers when we work on the tiers project. But it's probably going to be more complicated to use the `Upgrade` word overall.

Reply:
Updated